### PR TITLE
feat: integrate sider and header

### PR DESCRIPTION
**Pull request by danialdinh957**

### CODE REVIEW

**Comment by danialdinh957** [comment]
image
image

File: src/layouts/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import type { PropsWithChildren, ReactNode } from 'react'
+import { Layout } from '@ancient8/components'
+
+function MainLayout({children}: PropsWithChildren) {
+  return (
+    <Layout
+      config={{
+        sider:{
+          baseUrl: 'https://cms-dev.ancient8.gg/api/side-menus'
```
Comment (linhha796) [inline src/layouts/index.tsx:9]: use env variable

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,17 +1,78 @@
...
+}
+
+const supportedChains = [sepolia, ancient8] as [
+  RainbowKitChain,
+  ...RainbowKitChain[],
+]
+
+const wagmiConfig = getDefaultConfig({
+  appName: 'a8-super-app',
+  projectId: '2c6406174410a7ddd96218ba9a0f4476',
```
Comment (linhha796) [inline src/pages/_app.tsx:44]: use env

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -1,17 +1,78 @@
 import type { AppProps } from 'next/app'
+import type { NextPage } from 'next'
+import type { ReactElement, ReactNode } from 'react'
 
 import QueryClientProvider from '@/providers/QueryClientProvider'
 import ThemeProvider from '@/providers/ThemeProvider'
 
+import {
```
Comment (linhha796) [inline src/pages/_app.tsx:8]: move this group onto top

**Pull request merged**